README.md

```diff
@@ -334,6 +334,21 @@ You can configure the OpenTelemetry SDK for applications which can't currently b
 instrumentation.opentelemetry.io/inject-sdk: "true"
 ```
 
+#### Controlling Instrumentation Capabilities
+
+The operator allows specifying, via the feature gates,  which languages the Instrumentation resource may instrument.
+These feature gates must be passed to the operator via the `--feature-gates` flag.
+The flag allows for a comma-delimited list of feature gate identifiers.
+Prefix a gate with '-' to disable support for the corresponding language.
+Prefixing a gate with '+' or no prefix will enable support for the corresponding language.
+If a language is enabled by default its gate only needs to be supplied when disabling the gate.
+
+| Language | Gate                                  | Default Value   |
+|----------|---------------------------------------|-----------------|
+| .NET     | `operator.autoinstrumentation.dotnet` | enabled         |
```

Comment:
how is it disabled? with `disabled` value?

Reply:
iirc you can do `--feature-gate=-operator.autoinstrumentation.dotnet`

Reply:
@pavolloffay I added the disabling instructions above the table:

> Prefix a gate with '-' to disable support for the corresponding language.

Let me know if I need to tweak the paragraph or table to make things more obvious.